src/lib/ElementDescriptorContainer.js

```diff
@@ -61,7 +62,6 @@ import BezierCurveToDescriptor from './descriptors/Geometry/Shapes/BezierCurveTo
 import QuadraticCurveToDescriptor from './descriptors/Geometry/Shapes/QuadraticCurveToDescriptor';
 import AbsArcDescriptor from './descriptors/Geometry/Shapes/AbsArcDescriptor';
 import AbsEllipseDescriptor from './descriptors/Geometry/Shapes/AbsEllipseDescriptor';
-import HoleDescriptor from './descriptors/Geometry/Shapes/HoleDescriptor';
```

Comment:
@toxicFork I intentionally removed it below. It was commented out and triggering Eslint. If it is intended to be included I can uncomment it here.